upgrade_analysis/compare.py

```diff
@@ -237,7 +241,7 @@ def match(match_fields, report_fields, warn=False):
         return count
 
     matched_direct = match(
-        ["module", "mode", "model", "field"],
+        ["module", "model", "field", "type"],
```

Comment:
Why removing "mode"?

Reply:
OK, I don't know the historical reasons to be there, and I was thinking in selection fields where mode may be "modify" or similar.

Reply:
![Selection_4471](https://github.com/user-attachments/assets/2c721a90-290f-48ad-b106-df1dad24d553)

BTW, I realized why was done that way, the matches. It happens that "type" is filled only when mode is "create", but not when mode is "write". I readjusted the matches once I figured this, and the analysis is fine. I will push it soon.